src/cargo/ops/cargo_new.rs

```diff
@@ -99,6 +166,15 @@ fn existing_vcs_repo(path: &Path, cwd: &Path) -> bool {
     GitRepo::discover(path, cwd).is_ok() || HgRepo::discover(path, cwd).is_ok()
 }
 
+fn file(p: &Path, contents: &[u8]) -> io::Result<()> {
+    try!(File::create(p)).write_all(contents)
+}
+
+#[allow(deprecated)] // connect => join in 1.3
+fn join_licenses(v: Vec<String>) -> String {
+    v.connect("/")
```

Comment:
Probably for 1.2 compatiblity. I think we dropped that, so it's no longer necessary.